app/assets/javascripts/components/ClientList.jsx

```diff
@@ -0,0 +1,23 @@
+class ClientList extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			clients: [],
+		};
+	}
+
+	componentDidMount() {
+		Requester.get('/api/clients').then((clients) => {
+			this.setState({ clients });
```

Comment:
shouldn't this be this.setState({ clients: clients });?

Reply:
with es6 those statements are equivalent as long as the object key has the same name as the variable name

Reply:
maybe it would be clearer to not use the shorthand though?